src/assets/scss/index.scss

```diff
@@ -7,11 +7,12 @@
 
 @import 'components/_responsive.scss';
 
-@import 'components/_base.scss'; 
 
 
 
 @import 'components/_header.scss';
 
 @import 'components/_body.scss';
 @import 'components/footer.scss';
+
+@import 'components/_base.scss'; 
```

Comment:
@vihan85 
tại sao base lại nằm ở dưới cùng ?

Reply:
Hi @huulochuynh 
This issue has been resolved

Thanks best & regards